overlay.d/05rhcos/usr/lib/dracut/modules.d/41rhcos-check-luks-syntax/rhcos-fail-boot-for-older-luks-syntax

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script tries to look for a `/etc/clevis.json` reference 
+# (used to enable LUKS in RHCOS 4.6 or earlier versions) in an 
+# Ignition config. If it exists and the version of RHCOS is 4.7, 
+# then the script ends with exit 1.
+
+ignition_cfg="/run/ignition.json"
+wanted_path="/etc/clevis.json"
+tmpconfig=${outputdir}/tmpconfig.json
+
+# select the `/etc/clevis.json` entry from a given Ignition config
+(jq ".storage.files[] | select(.path==\"${wanted_path}\")") \
+    < "${ign_config}" > ${tmpconfig}.fragment
+
+if [ -s ${tmpconfig}.fragment ]; then
```

Comment:
Hmm, I think we can avoid a tmpconfig entirely and just use `jq -e`. E.g. something like:

```bash
if jq -e ".storage.files[] | select(.path==\"${wanted_path}\")"; then
    ...
fi
```

Reply:
> Hmm, I think we can avoid a tmpconfig entirely and just use `jq -e`. E.g. something like:
> 
> ```shell
> if jq -e ".storage.files[] | select(.path==\"${wanted_path}\")"; then
>     ...
> fi
> ```
@jlebon  I think we should provide an input i.e. `ign_config` in this case, right?